### DISCUSSION TITLE
svg export does not work

### DISCUSSION THREAD

**Discussion by Cristobal-84** [friction2d/friction#133] [Q&A]
Hello,
Can someone tell me why the SVG export of this simple animation does not work ?
[Essai.zip](https://github.com/friction2d/friction/files/14810878/Essai.zip)

**Comment by rodlie** [comment]
You are using transform effects.

**Reply by Cristobal-84** [reply]
no, I use a follow path effect
it works when I move the cursor in the time bar or when I play but not when exporting
